source/common/common/assert.h

```diff
@@ -25,6 +27,7 @@
  * Indicate a panic situation and exit.
  */
 #define PANIC(X)                                                                                   \
+  BACKTRACE_PROD_LOG();                                                                            \
```

Comment:
Maybe remove these from this PR and wait until the signal handler PR one so that we can just trigger on `sigabrt`.